src/mainwindow.cpp

```diff
@@ -197,6 +197,7 @@ void MainWindow::tableOfContentToggled(bool state)
     if (state) {
         checkActionNoSignal(KiwixApp::ToggleReadingListAction, false);
         mp_ui->sideBar->setCurrentWidget(mp_ui->tableofcontentbar);
+        mp_ui->sideBar->setStyleSheet("background-color: white;");
```

Comment:
This is a bug. If the state has not changed, the stylesheet won't change. Like image below.

You should be able to change the background color like TOC, try to investigate the CSS changes before doing ad-hoc changes like this.

![Screenshot from 2024-12-02 14-44-11](https://github.com/user-attachments/assets/0cee9981-a201-43fb-a4b1-73885fd115a5)